### ISSUE TITLE
Bro really thought no one would notice 😂😂

### ISSUE THREAD

**Issue by coopers-lab** [burger2281/TikTok-Viewbot#1]
Don't run this, it tries to install malware on your computer but because this guy is like 13 he messed it up and it doesn't work...

Neither does the viewbot btw, even after you remove the attempted malware...

**Comment by MrFlooky** [comment]
i didn't notice😂
but it really doesn't work, so...

**Comment by coopers-lab** [comment]
Yeah there’s a few lines after the imports that does it in a way that antivirus couldn’t pick it up without being compiled… Smart of him, cause most people (including me) would run it without checking the file…
Luckily there was an error which made me suspicious and I checked it…

**Comment by aust-ahuman** [comment]
Fr my friend ran it didn't end good for him thankfully he got the virus off in time
